lib/fungus_toast/games/surrounding_cells.ex

```diff
@@ -0,0 +1,3 @@
+defmodule FungusToast.Games.SurroundingCells do
+  defstruct top_left_cell: nil, top_cell: nil, top_right_cell: nil, right_cell: nil, bottom_right_cell: nil, bottom_cell: nil, bottom_left_cell: nil, left_cell: nil
```

Comment:
Do we really need to do this? Why not just do a list or map in the GridCell? You can just say position 0 is always top left or something like that, or just store it in a map with these as keys

Reply:
This is also in the category of "I'm sure you're right", but I need someone to help me through the right way to do this in Elixir. In this case, the high level question is "when should I use a struct vs. just dynamically setting this stuff in a map" like you're suggesting?

Reply:
I'm not quite sure either, it just seems odd to pull this out into its own file. @dshah would probably be the better person to ask

Reply:
I also may have jumped the gun. I thought this was a full model with a schema and now see that it is not

Reply:
Yeah, i changed this to a map, but I may turn it back to a struct. From what I can tell, using structs whenever you can seems to be a good thing. Seems like it will help with serialization too.